pkg/gui/custom_commands.go

```diff
@@ -151,10 +153,65 @@ func (gui *Gui) handleCustomCommandKeybinding(customCommand config.CustomCommand
 						return gui.surfaceError(err)
 					}
 
+					return gui.createMenu(title, menuItems, createMenuOptions{showCancel: true})
+				}
+			case "menuFromCommand":
+				f = func() error {
+                                        // Collect cmd to run from config
+			                cmdStr, err := gui.resolveTemplate(prompt.Command, promptResponses)
+			                if err != nil {
+			                	return gui.surfaceError(err)
+			                }
+
+                                        // Collect Filter regexp
+			                filter, err := gui.resolveTemplate(prompt.Filter, promptResponses)
+			                if err != nil {
+			                	return gui.surfaceError(err)
+			                }
+
+                                        // Run and save output
+                                        message,err := gui.GitCommand.RunCommandWithOutput(cmdStr)
+			                if err != nil {
+			                	return gui.surfaceError(err)
+			                }
+
+					// Need to make a menu out of what the cmd has displayed
+                                        var candidates []string
+                                        reg := regexp.MustCompile(filter)
+                                        for _,str := range strings.Split(string(message), "\n"){
+                                            cand := ""
+                                            if str != "" {
+					        for i := 1; i < (reg.NumSubexp()+1); i++ {
+                                                    keep := reg.ReplaceAllString(str, "${"+fmt.Sprint(i)+"}")
```

Comment:
For converting numbers to strings [strconv. Itoa()](https://pkg.go.dev/strconv@master#Itoa) is a better solution